integration_tests/features/steps/fal_steps.py

```diff
@@ -15,6 +21,69 @@ def run_command_step2(context, command):
     _run_command(command)
 
 
+@given("the project 001_flow_run_with_selectors")
+def set_project_folder(context):
+    context.base_dir = reduce(
+        os.path.join, [os.getcwd(), "projects", "001_flow_run_with_selectors"]
+    )
+    context.temp_dir = tempfile.TemporaryDirectory()
+    os.environ["temp_dir"] = context.temp_dir.name
+
+
+@when("fal flow is invoked with following arguements")
```

Comment:
arguments*

Reply:
Last time we had a pr for this sort of stuff, we had settled on 
```
When `fal run` is run
```

https://github.com/fal-ai/fal/blob/main/integration_tests/features/run.feature#L3

why not reuse that style?

Reply:
I will just do this: 

```
  Scenario: fal flow run command with selectors
    Given the project 001_flow_run_with_selectors
    When the following command is invoked:
    """
    fal flow run --profiles-dir $baseDir --project-dir $baseDir --select before.py+
    """
    Then the following models are calculated:
    | agent_wait_time |
    And the following scripts are ran:
    | agent_wait_time.before.py | agent_wait_time.after.py |
```